lib/bolt_server/transport_app.rb

```diff
@@ -271,6 +287,10 @@ def pe_task_info(pal, module_name, task_name, environment)
       }
     end
 
+    def task_list(pal)
+      pal.list_tasks.map { |task_name, _description| { 'name' => task_name } }
```

Comment:
This needs to be updated to filter tasks based on allowlist in bolt-project.yaml

Reply:
Updated.